### ISSUE TITLE
invalid identifier `DRETURN-1` generated by `opeo:decompile`

### ISSUE THREAD

**Issue by yegor256** [objectionary/opeo-maven-plugin#411]
I see this inside XMIR, after running `opeo:decompile`:

```
<o base="opcode" line="999" name="DRETURN-1">
  <o base="int" data="bytes" line="495871679">00 00 00 00 00 00 00 AF</o>
</o>
```

The `DRETURN-1` is not a valid identifier in EO. All identifiers must start with a small English letter.

**Comment by yegor256** [comment]
@volodya-lombrozo please, help with this one

**Comment by 0pdd** [comment]
@yegor256 the puzzle [#419](https://github.com/objectionary/opeo-maven-plugin/issues/419) is still not solved.

**Comment by volodya-lombrozo** [comment]
@rultor release, tag is `0.3.5`, title is `Version 0.3.5`.

**Comment by volodya-lombrozo** [comment]
@yegor256 Done. This should be fixed in the `0.3.5` version.

**Comment by yegor256** [comment]
@volodya-lombrozo thanks, fixed!
